Makefile.kokkos

```diff
@@ -1187,6 +1205,49 @@ ifeq ($(KOKKOS_INTERNAL_USE_HPX), 1)
   KOKKOS_TPL_LIBRARY_NAMES += hpx
 endif
 
+ifeq ($(KOKKOS_INTERNAL_USE_TBB), 1)
+  KOKKOS_SRC += $(wildcard $(KOKKOS_PATH)/core/src/TBB/*.cpp)
+  KOKKOS_HEADERS += $(wildcard $(KOKKOS_PATH)/core/src/TBB/*.hpp)
+  ifneq ($(TBB_PATH),)
+    ifeq ($(KOKKOS_INTERNAL_ENABLE_DEBUG), 1)
+      #KOKKOS_CXXFLAGS += $(shell PKG_CONFIG_PATH=$(TBB_PATH)/lib64/pkgconfig pkg-config --cflags tbb_debug)
+      #KOKKOS_CXXLDFLAGS += $(shell PKG_CONFIG_PATH=$(TBB_PATH)/lib64/pkgconfig pkg-config --libs tbb_debug)
+      #KOKKOS_LDFLAGS += $(shell PKG_CONFIG_PATH=$(TBB_PATH)/lib64/pkgconfig pkg-config --libs tbb_debug)
+      KOKKOS_CXXFLAGS += -DTBB_USE_GLIBCXX_VERSION=70401 -I/cvmfs/cms.cern.ch/slc7_amd64_gcc700/external/tbb/2019_U8/include
+      KOKKOS_LDFLAGS += -L/cvmfs/cms.cern.ch/slc7_amd64_gcc700/external/tbb/2019_U8/lib
```

Comment:
Here and below, should the uncommented lines be removed (as specific to CMS setup) and commented lines be uncommented?

Reply:
I have no knowledge of pkg-config so have no idea what would be the correct names to use

Reply:
I followed what was done for `QTHREADS` which also does not have a pkg-config.